README.md

```diff
@@ -47,7 +47,17 @@
 ```
 
 ```csharp
-// Coming Soon
+using Bandwidth.Standard.Messaging;
+using Bandwidth.Standard.Messaging.Models;
+using Bandwidth.Standard.Messaging.Controllers;
+
+MessageRequest msgRequest = new MessageRequest();
+msgRequest.ApplicationId = applicationId;
+msgRequest.From = "+18888888888";
+msgRequest.To = new string[1] {"9199199999"};
+msgRequest.Text = "The quick brown fox jumps over a lazy dog.";
+
+msgController.CreateMessage(msgUserId, msgRequest);
```

Comment:
For consistency can you either change the user id to look like the example on line 101, or change the one on line 101 to look like this?

Reply:
`msgController.CreateMessage(accountId, msgRequest);`